Alignment/MuonAlignment/plugins/MuonMisalignedProducer.cc

```diff
@@ -84,7 +84,7 @@ MuonMisalignedProducer::MuonMisalignedProducer(const edm::ParameterSet& p)
       esTokenGEM_(esConsumes(edm::ESInputTag("", "idealForMuonMisalignedProducer"))) {}
 
 //__________________________________________________________________________________________________
-MuonMisalignedProducer::~MuonMisalignedProducer() = default;
+MuonMisalignedProducer::~MuonMisalignedProducer() {}
```

Comment:
I think the `= default;` is the recommended way of doing this

Reply:
The code-format tool changed this. do you want to keep ```= default;```?

Reply:
Interesting, @makortel @smuzaffar I thought the  `default` was the preferred option

Reply:
> Hi @hyunyong I wrote to you privately, you did many things in the PR from which some are correct and some are undoing the work of @mmusich about the DB output module changes

Oh, ok. 
I copied the module from an old version.
I understood now what happened.
I will close this PR and make a new one.

Reply:
It is, and I'm a bit surprised that code-checks would change away from `= default`. We have plenty of those in `.cc` files.

Reply:
> It is, and I'm a bit surprised that code-checks would change away from `= default`. We have plenty of those in `.cc` files.

no, it was my mistake.